data/network/src/main/java/io/filmtime/data/network/TmdbMoviesService.kt

```diff
@@ -9,13 +9,11 @@ interface TmdbMoviesService {
 
   @GET("movie/{movie_id}")
   suspend fun getMovieDetails(
-    @Path("movie_id") movieId: Int,
-    @Query("api_key") apiKey: String = BuildConfig.TMDB_API_KEY,
+    @Path("movie_id") movieId: Int
```

Comment:
Please do not remove trailing commas
```suggestion
    @Path("movie_id") movieId: Int,
```